src/main.ts

```diff
@@ -1,8 +1,13 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 
+import { config } from 'dotenv';
```

Comment:
```suggestion
import { NestFactory } from '@nestjs/core';
import { config } from 'dotenv';

import { AppModule } from './app.module';
```

로컬 패키지와 설치된 패키지는 구분해야 합니다